css_02/css_02.css

```diff
@@ -0,0 +1,122 @@
+html *
+{
+   font-family: Arial, Helvetica, sans-serif !important;
+}
+
+.header 
+{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 91%;
+    height: 40px;
+}
+
+.greenButtons 
```

Comment:
In the previous CSS exercise we were discussing the different type of naming conventions. You may want to look into http://bdavidxyz.com/blog/how-to-name-css-classes/ section 5 and 6. I request no changes currently - just please read the article, it will be helpful for the project / future.